ext/mysqlnd/mysqlnd_wireprotocol.c

```diff
@@ -267,6 +267,24 @@ mysqlnd_read_header(MYSQLND_PFC * pfc, MYSQLND_VIO * vio, MYSQLND_PACKET_HEADER
 		pfc->data->packet_no++;
 		DBG_RETURN(PASS);
 	}
+	// @see https://dev.mysql.com/worklog/task/?id=12999
+	if (header->size > 0) {
+		DBG_ERR_FMT("Logical link: try reap pending data. Packet size=%zu", header->size);
```

Comment:
I understand you added this to be consistent with the rest of the code, but I am not sure that we need it.

Reply:
OK